docs/release-notes/8.17.asciidoc

```diff
@@ -9,6 +9,22 @@
 [[known-issue-8.17.1]]
 ==== Known issues
 
+// tag::known-issue[]
+[discrete]
+.{elastic-sec} crashes on {kib} instances with 1 GB of RAM 
```

Comment:
Should we clarify that this happens on Elastic Cloud instances? Maybe something like that:

```suggestion
.{elastic-sec} crashes on {kib} instances with 1 GB of RAM in Elastic Cloud deployments
```

Reply:
Yeah, that's a good idea. I'll add this to the other place where this known issue is doc'd.